tests/test_esm.py

```diff
@@ -10,19 +10,33 @@ class ESMTest(UbuntuAdvantageTest):
 
     def test_enable_esm(self):
         """The enable-esm option enables the ESM repository."""
-        process = self.script('enable-esm', 'user:pass')
-        self.assertEqual(0, process.returncode)
-        self.assertIn('Ubuntu ESM repository enabled', process.stdout)
-        expected = (
+        expected_repo_list = {}
+        expected_repo_list['precise'] = (
             'deb https://user:pass@esm.ubuntu.com/ubuntu precise main\n'
             '# deb-src https://user:pass@esm.ubuntu.com/ubuntu precise main\n')
-        self.assertEqual(expected, self.repo_list.read_text())
-        keyring_file = self.trusted_gpg_dir / 'ubuntu-esm-keyring.gpg'
-        self.assertEqual('GPG key', keyring_file.read_text())
-        # the apt-transport-https dependency is already installed
-        self.assertNotIn(
-            'Installing missing dependency apt-transport-https',
-            process.stdout)
+        expected_repo_list['trusty'] = (
+            'deb https://user:pass@esm.ubuntu.com/ubuntu '
+            'trusty-security main\n'
+            '# deb-src https://user:pass@esm.ubuntu.com/ubuntu '
+            'trusty-security main\n'
+            '\n'
+            'deb https://user:pass@esm.ubuntu.com/ubuntu '
+            'trusty-updates main\n'
+            '# deb-src https://user:pass@esm.ubuntu.com/ubuntu '
+            'trusty-updates main\n')
+        for series in ['precise', 'trusty']:
+            self.SERIES = series
+            process = self.script('enable-esm', 'user:pass')
+            self.assertEqual(0, process.returncode)
+            self.assertIn('Ubuntu ESM repository enabled', process.stdout)
+            self.assertEqual(expected_repo_list[series],
+                             self.repo_list.read_text())
+            keyring_file = self.trusted_gpg_dir / 'ubuntu-esm-keyring.gpg'
```

Comment:
os.path.join(self.trusted_gpg_dir, 'ubuntu-esm-keyring.gpg')

of course we will need something different for trusty

Reply:
I have another branch to add the unattended-upgrades config, it's https://github.com/panlinux/ubuntu-advantage-client/tree/oc-unattended-upgrades-pockets

I wanted to land things piece by piece.

About the gpg key, do we have it already, or should I just make one up and replace it when the time comes? I also wanted to do a separate branch for that.

Reply:
If you fix the keyring_file creation I'll +1 this.

Let's ask about the GPG key in standup tomorrow.

I'll take al ook at the UU config next.

Reply:
Ah, I see what you mean now about the keyring. Looks like I will have to handle that in this branch.